src/ocm/components/clusterConfiguration/OcmDiscoveryImageConfigForm.tsx

```diff
@@ -38,7 +43,9 @@ export interface OcmImageCreateParams {
   imageType?: OcmImageType;
 }
 
-export type OcmDiscoveryImageFormValues = OcmImageCreateParams & ProxyFieldsType;
+export type OcmDiscoveryImageFormValues = OcmImageCreateParams &
+  ProxyFieldsType &
+  TrustedCertificateFieldsType;
 
 const validationSchema = Yup.lazy<OcmDiscoveryImageFormValues>((values) =>
```

Comment:
As discussed, I think it's also important to let the user know which PEM format is accepted. Then they would know how to generate it, rather than finding out by trial and error.

Reply:
BE response: "I think it's best if we leave the validations in the backend for this one as it's an ugly format and I'm afraid parsing might be very complex in the UI, depending on how good the libraries available to do it are."

Reply:
If we are not going to have validations in the UI, we (or the BE) should at least provide the users with better error messages. The current ones aren't nice or helpful.

But that could be addressed in a new issue.

![image](https://user-images.githubusercontent.com/87187179/208381435-008982b0-de48-4874-b078-e4369804a346.png)
![image](https://user-images.githubusercontent.com/87187179/208381520-770d6c8a-3c5e-47db-ae37-f223edc52af0.png)

Reply:
Waiting for the response of BE about validations.

Reply:
I've opened a bug for BE to change validations messages: https://issues.redhat.com/browse/MGMT-13000

I've changed the title of the message to "Failed to create discovery image". We need to change the popover and we can merge it